hil/ext/switches/nexus.py

```diff
@@ -124,12 +124,22 @@ def disable_native(self, vlan_id):
     @staticmethod
     def connect(switch):
         """Connect to the switch."""
+        alternatives = ['[Pp]assword: ', '>', '#']
         console = pexpect.spawn(
             'ssh ' + switch.username + '@' + switch.hostname)
 
-        console.expect('Password: ')
-        console.sendline(switch.password)
+        outcome = console.expect(alternatives)
+        if outcome == 0:
+            console.sendline(switch.password)
+            outcome = console.expect(alternatives)
+        if outcome == 1:
+            console.sendline('en')
```

Comment:
It's probably worth factoring this out into a helper function, and using it in both places.

Reply:
done.